spec/validators/future_date_validator_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+class Validatable
+  include ActiveModel::Validations
+  attr_accessor :date
+  validates :date, future_date: true
+end
+
+RSpec.describe FutureDateValidator do
+  subject { Validatable.new }
+
+  context "when date is nil" do
+    it "is invalid" do
+      expect(subject).to be_invalid
+    end
+
+    it "adds error message about nil" do
+      subject.validate
```

Comment:
For me it looks better without breaks. I think you shouldn't break method body until it's necessary and here there are only 2 lines so I don't think it needs any breaks.